requirements.txt

```diff
@@ -1,4 +1,5 @@
-mkdocs-multirepo-plugin==0.7.0
-mkdocs-material==9.5.14
-mike==2.1.0
+mkdocs-multirepo-plugin==0.8.3
+mkdocs==1.6.1
```

Comment:
mkdocs rev to remove warning about binding to 0.0.0.0 - without, we'd not be able to run mkdocs strict builds (to catch broken links etc)